dscblog/pages/info/footer.html

```diff
@@ -0,0 +1,237 @@
+{% block title %}
+    <meta charset="UTF-8" />
+    <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+    <script defer src="https://use.fontawesome.com/releases/v5.0.13/js/all.js" integrity="sha384-xymdQtn1n3lH2wcu0qhcdaOpQwyoarkgLVxC/wZ5q7h9gHtxICrpcaSUfygqZGOe" crossorigin="anonymous"></script>
```

Comment:
don't use this library since we are not using it elsewhere. Instead, put the icons in the `static/icons` directory.